travelwise/travelwise/settings.py

```diff
@@ -32,6 +32,8 @@
 
 CSRF_TRUSTED_ORIGINS = os.environ.get("DJANGO_CSRF_TRUSTED_ORIGINS").split(" ")
 
+# HTTPS Redirect
+SECURE_SSL_REDIRECT = True if not DEBUG else False
```

Comment:
`not DEBUG`?

Reply:
Debug = 1 // Development
Debug = 0 // Production

Reply:
I understand that we probably don't want SSL enabled during development. 
I meant why not just `not DEBUG`, since the `not` operator should already return `True` for truthy values and `False` for falsy values?

Reply:
YOU RIGHT!!!

Reply:
Fixed